src/Migrations/M032MigrateSocialMediaTwitterBlockToEmbedBlock.php

```diff
@@ -0,0 +1,360 @@
+<?php
+
+// phpcs:disable Generic.Files.LineLength.MaxExceeded
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+use WP_Block_Parser;
+
+/**
+ * Migrate Social Media blocks with Twitter/X urls to Embed blocks.
+ */
+class M032MigrateSocialMediaTwitterBlockToEmbedBlock extends MigrationScript
+{
+    private const SOURCE_INSTAGRAM = 'instagram';
+    private const SOURCE_FACEBOOK = 'facebook';
+    private const SOURCE_TWITTER = 'twitter';
+    private const SOURCE_X = 'x';
+
+    private const SOURCE_PATTERNS = [
```

Comment:
Should we double-check whether these services use alternative domains as YouTube does (for example, youtube.com, m.youtube.com, youtu.be, etc.)?